prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullname = ['Nolan Reda'];
+    console.log(fullname);
+
+    var numberOfPets = ['2'];
+    console.log(numberOfPets);
```

Comment:
Hi Nolan! Here you are creating arrays with the `[]` square brackets, you can remove those.

for your `numberOfPets` variable, be sure to also remove the quotation marks so that the JavaScript engine doesn't interpret your variable as a string.